packages/react-storage/package.json

```diff
@@ -66,7 +66,7 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "64.13 kB",
+      "limit": "64.19 kB",
```

Comment:
Is the same value as the bump on `main`?

Reply:
main is still `64.13` maybe the eslint changes help reduce this. Once we merge back to main i'll try to bring this back down

Reply:
Discussed offline. For posterity more concerned abt the git history than the actual value